charts/dealer/templates/dealer-deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "dealer.fullname" . }}
+  labels:
+    app: {{ template "dealer.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "dealer.fullname" . }}
+      release: {{ .Release.Name }}
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "dealer.fullname" . }}
+        release: "{{ .Release.Name }}"
+      annotations:
+        prometheus.io/path: /metrics
+        prometheus.io/port: "3333"
+        prometheus.io/scrape: "true"
+      {{- with .Values.annotations }}
+        {{ toYaml . | nindent 8 }}
+      {{- end }}
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.repository }}@{{ .Values.image.digest }}"
+        env:
+        - name: NETWORK
+          value: regtest
+        - name: LOG_LEVEL
+          value: debug
+        - name: ACTIVE_STRATEGY
+          value: OKEX_PERPETUAL_SWAP
+        - name: ACTIVE_WALLET
+          value: REMOTE_WALLET
+        - name: OKEX5_KEY
+          value: key
```

Comment:
What is this setting for?

Reply:
ACTIVE_WALLET env. var. was so it could be switched between the remote (currently fake) wallet and the simulated version.
Simulated wouldn't make sense outside testing anyways so maybe default to remote in the code if env. var. is missing?